scanner/cmd/updater/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+
+	"github.com/quay/zlog"
+	"github.com/stackrox/stackrox/scanner/v4/updater"
+)
+
+func main() {
+	// Parse command-line flags
+	outputDir := flag.String("outputDir", "", "Output directory")
```

Comment:
I think something like `output-dir` is usually preferred over `outputDir` for long names

Reply:
<img width="792" alt="Screenshot 2023-07-28 at 5 13 40 PM" src="https://github.com/stackrox/stackrox/assets/5618748/46059fab-0e9f-4661-9c49-9621d05662ea">

Reply:
I was referring to the flag name, not the variable name